build_tools/benchmarks/README.md

```diff
@@ -0,0 +1,49 @@
+# IREE Benchmark Suites Tool
+
+This directory contains the tools to run IREE benchmark suites and generate
+reports. More information about benchmark suites can be found [here](benchmarks/README.md).
+
+## Benchmark Tools
+Currently we have `run_benchmarks_on_android.py` and `run_benchmarks_on_linux.py` to run
+benchmark suites on Android devices (with `adb`) and Linux local machines.
```

Comment:
grammar/style nits:

* newline between header and content
* add "scripts" ("files" would also work)
* drop "local" (doesn't really add much to qualify, when we benchmark on cloud machines anyways)

```suggestion
## Benchmark Tools

Currently we have `run_benchmarks_on_android.py` and `run_benchmarks_on_linux.py`
scripts to run benchmark suites on Android devices (with `adb`) and Linux machines.
```